### DISCUSSION TITLE
上传部分文件时，报错上传失败, 服务异常，请稍后再试

### DISCUSSION THREAD

**Discussion by znc15** [lsky-org/lsky-pro#741] [问答]
日志：https://paste.ubuntu.com/p/tH5FF8NPZd/
环境：
宝塔面板
PHP:8.1.30
nginx:1.21.4
Mysql:5.7.40

上传成功非常随机
大概觉得是imgck的问题，读不到路径，但是不知道怎么弄

**Comment by heiyuan0801** [comment]
lskypro.com  可以反馈一下问题
